README.md

```diff
@@ -1 +1,46 @@
-# hello-project
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Project Name
+Hello Project
+
+> Description the project.
+It is done with html and css
+
+## Built With 
+
+- Major languages: HTML and CSS
+- Technologies used: Modified with linters 
+
+## Live Demo (if available)
+
+[Live Demo Link](https://livedemo.com)
+
+
+## Authors
```

Comment:
- [ ] Kindly add the missing section `Getting Started` before `Authors`, that section is supposed to tell the users how to setup the project locally and run it. Please use Microverse's [template](https://github.com/microverseinc/readme-template). You can mention that users need a code editor like vscode, a browser like chrome... and you can also guide them through cloning your project and opening `index.html` in the browser.. etc. 👍